Project/Source/StateMachineManager.cpp

```diff
@@ -17,54 +16,65 @@ bool StateMachineManager::Contains(std::list<AnimationInterpolation>& animationI
 	return false;
 }
 
-void StateMachineManager::SendTrigger(const std::string& trigger, std::unordered_map<UID, float>& currentTimeStates, std::list<AnimationInterpolation>& animationInterpolations, const UID& stateMachineResourceUID, State& currentState, State& currentStateSecondary, std::unordered_map<UID, float>& currentTimeStatesPrincipal, StateMachineEnum stateMachineEnum, std::list<AnimationInterpolation>& animationInterpolationsSecondary) {
+void StateMachineManager::SendTrigger(const std::string& trigger, StateMachineEnum stateMachineSelected, ComponentAnimation& componentAnimation) {
+	UID stateMachineResourceUID = componentAnimation.stateMachineResourceUIDPrincipal;
```

Comment:
Could this block (from line 20 to line 31) be simplified doing the ternary operator?

Ex: UID stateMachineResourceUID = (stateMachineSelected == StateMachineEnum::SECONDARY) ? componentAnimation.stateMachineResourceUIDSecondary : componentAnimation.stateMachineResourceUIDPrincipal;